index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Identities
```

Comment:
I personally feel that "digital identities" are inclusive of "federated identities", but just wanted to make sure that that's the case, because, if not, then, maybe we'd need to use a more general term (i.e. remove the "digital" prefix)?

My intuition is that "digital identities" is inclusive and a good starting point, but just wanted to be transparent of how I'd want to see this spec evolve over time.

Reply:
Tbh, I think we might go through a few more renames over the next few weeks.

Reply:
Agreed. I think Digital Identities is a great starting point. Resolving.